workshops/build-a-binderhub/workshop-presentations/zero-to-binderhub.md

```diff
@@ -197,43 +198,63 @@ Do this **before** adding any secrets to the folder!
 
 Adapted from [Step Zero: Kubernetes on Microsoft Azure Container Service (AKS)](https://zero-to-jupyterhub.readthedocs.io/en/latest/microsoft/step-zero-azure.html).
 
-A short (and by no means exhaustive) [glossary](#glossary-of-kubernetes-terms) of Kubernetes terms is given at the end of this workshop, should you require further explanation.
+A short (but by no means exhaustive) [glossary](#glossary-of-kubernetes-terms) of Kubernetes terms is given at the end of this workshop, should you require further explanation.
 
-**NOTE:** You shouldn't need to login to Azure when using the Cloud Shell as it reads your login from the Portal.
+#### 1. Login to your account
 
-#### 1. Activate your Subscription
+Run the following command:
 
-To see a list of Azure subscriptions you have available to you, you can run the following command.
+```bash
+az login
+```
+
+You should then see a message reading:
+
+```bash
+To sign in, use a web browser to open the page https://microsoft.com/devicelogin and enter the code A-RANDOM-CODE to authenticate.
 ```
+
+Visit the webpage <https://microsoft.com/devicelogin> and enter the random code as it appears in your cloud shell.
+Then select the account that you would like to sign in with.
+
+#### 2. Activate your Subscription
+
+To see a list of Azure subscriptions you have available to you, you can run the following command.
+
+```bash
 az account list --refresh --output table
 ```
+
 This prints your subscriptions to the terminal in a human-readable format.
 Now let's set our working subscription.
 
 **If you are using a Free Trial suscription:**
-```
-az account set -s "Free Trial"
+
+```bash
+az account set --subscription "Free Trial"
 ```
 
 **If you are using an Azure Pass:**
-```
-az account set -s "Azure Pass - Sponsorship"
+
+```bash
+az account set --subscription "Azure Pass - Sponsorship"
 ```
 
 :vertical_traffic_light: :vertical_traffic_light: :vertical_traffic_light: :vertical_traffic_light: :vertical_traffic_light:
 
 **NOTE:** If your subscription name has no whitespace, the quotation marks are not required.
 
-#### 2. Create a Resource Group
+#### 3. Create a Resource Group
 
 Resource Groups are how the Azure environment labels services that are related to each other (further details in [this blog post](http://www.onlinetech.com/resources/references/how-to-use-azure-resource-groups-a-simple-explanation)).
 We will create a resource group in a specific region and create computational resources _within_ this group.
 
-```
+```bash
 az group create --name testhub \
     --location westeurope \
     --output table
 ```
+
 * `--name` specifies the name of your resource group and should be something that uniquely identifies this hub.
```

Comment:
Potential source of confusion? "uniquely identifies this hub".  Multiple people in a workshop will be using the same name `testhub` which suggests you do not need to have a unique name. I do not know anything about Azure so I am not sure. A possible re-wording:
"...name of your resource group. In practice this should be something that uniquely identifies this hub, but for this workshop, we will all use the same name `testhub`"